src/docfx/Errors.cs

```diff
@@ -457,11 +457,39 @@ public static Error MonikerOverlapping(string uid, List<FilePath> files, IEnumer
                 => new Error(ErrorLevel.Error, "moniker-overlapping", $"Two or more documents with the same uid `{uid}`({StringUtility.Join(files)}) have defined overlapping moniker: {StringUtility.Join(overlappingMonikers)}.");
 
             /// <summary>
-            /// Failed to parse moniker string.
+            /// Failed to parse moniker string: moniker is not defined.
             /// </summary>
             /// Behavior: ✔️ Message: ❌
-            public static Error MonikerRangeInvalid(SourceInfo? operand, FormattableString message)
-                => new Error(ErrorLevel.Error, "moniker-range-invalid", message, operand);
+            public static Error MonikerRangeMissing(SourceInfo<string?> operand, string moniker)
```

Comment:
@928PJY to see if this one needs to be expanded to multiple error codes. I am tentative to do this change. This error is like regular expression parse error, we could have a single exception type, with exception details as args, like this:
```suggestion
            public static Error MonikerRangeInvalid(SourceInfo source, string message)
```